packages/back-end/src/integrations/BigQuery.ts

```diff
@@ -78,4 +81,46 @@ export default class BigQuery extends SqlIntegration {
   castUserDateCol(column: string): string {
     return `CAST(${column} as DATETIME)`;
   }
+
+  async getInformationSchema(
+    dataSourceType: DataSourceType,
+    projectId: string
+  ): Promise<InformationSchema[] | null> {
+    if (!projectId) {
+      return null;
+    }
+
```

Comment:
Eventually, we'll probably want to update this to take in an array of projectIds, and map over that array to build the list of datasets.